resources/core/charts/k8s-dashboard-proxy/templates/virtual-service.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
```

Comment:
What is the reason for creating another gateway and listening on HTTP not HTTPS?

Reply:
I'll double check 👍

Reply:
You're right! I removed it. Thanks!